.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/dapphub/ds-test
 [submodule "lib/semaphore"]
 	path = lib/semaphore
-	url = https://github.com/appliedzkp/semaphore
+	url = https://github.com/cichaczem/semaphore
```

Comment:
This absolutely needs to be forked into the worldcoin organisation, perhaps as `semaphore-v3` or by bumping the existing fork.

Reply:
Yup, I didn't want to wait for DevOps so just forked it myself.